text/0000-add-status-field.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Add Status to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
```

Comment:
```suggestion
- Status: Implementable <!-- Acceptable values: Implementable, Implemented, On Hold, Superseded -->
```

We've discussed using Implementable and Implemented in other conversations.

Reply:
@ekcasey @natalieparellano @natalieparellano 

I'm sorry for not being present when these values were posited. Can I please just confirm that the following acceptable values are what's preferred?

```
Draft, Implementable, Implemented, On Hold, Superseded
```

Reply:
@ekcasey @natalieparellano @hone 

I'm sorry for not being present when these values were posited. Can I please just confirm that the following acceptable values are what's preferred?

```
Draft, Implementable, Implemented, On Hold, Superseded
```

Reply:
I like the idea in general of indicating to users whether an RFC has been implemented, but I anticipate amibiguity regarding the definition of "implemented" as a status field. For example, image a change that adds a feature to the platform API is it implemented when the spec is released? when the lifecycle is released? when pack exposes it to users?

Given the ambiguity I suspect we would want something more detailed than a binary implementable/implemented distinction, like an implementation history section (example https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/970-kubeadm-config#implementation-history)?

If this is contentious or complicated though I think we could break this out into a different RFC and go ahead with a simple `Approved` status for now.

Reply:
@ekcasey, I agree. I can see that we'd want to define `Implemented`. For the sake of not ballooning this RFC, I'm more that okay with only moving forward with `Approved`.

Reply:
Does that mean that we agree on a total of:

```
Draft, Approved, On Hold, Superseded
```

Reply:
Well, _I_ agree.